music_detection/measure.py

```diff
@@ -0,0 +1,11 @@
+class Measure:
+    def __init__(self):
+        self.position_in_image = []
+        self.note_list = []
+        self.time_signature = []
```

Comment:
This should a number
```suggestion
        self.time_signature = 0
```